common/core/nesteddisplay.h

```diff
@@ -21,16 +21,73 @@
 #include <stdint.h>
 
 #include <memory>
-
 #include <nativedisplay.h>
+#ifdef NESTED_DISPLAY_SUPPORT
+#include <linux/hyper_dmabuf.h>
+#include <map>
+#include "drmbuffer.h"
+#include <utils/threads.h>
+#include <sys/socket.h>
+#include <netinet/in.h>
+#include "hwcthread.h"
+#include "hwctrace.h"
+#define SURFACE_NAME_LENGTH    64
+#endif
 
 namespace hwcomposer {
 
+struct HwcLayer;
+class NativeBufferHandler;
 class NestedDisplayManager;
 
+#ifdef NESTED_DISPLAY_SUPPORT
```

Comment:
We can remove this check ?

Reply:
I suggest keep this check, while in native android we don't need these data struct and the thread instance.